test/irmin/generic-key/test_store_offset.ml

```diff
@@ -140,12 +140,6 @@ module Slot_keyed_vector : Indexable.Maker_concrete_key1 = struct
         assert (k.slot < Vector.length t.data);
         Lwt.return_true)
 
-    let clear t =
-      let t = check_not_closed t in
-      Vector.clear t.data;
-      t.generation <- t.generation + 1;
```

Comment:
The `generation` field can now be dropped too, I think. 

Similarly, the `clear_since_add` and `clear_signal` fields in `./test_inlined_contents.ml` are no longer needed.